packages/eyeglass/src/functions/index.ts

```diff
@@ -1,16 +1,16 @@
-"use strict";
-
-import * as merge from "lodash.merge";
-
 import assetURI from "./asset-uri";
 import normalizeURL from "./normalize-uri";
 import version from "./version";
 import fs from "./fs";
+import { EyeglassFunctions } from "./EyeglassFunctions";
+import { FunctionDeclarations } from "../util/SassImplementation";
 
-export default function(eyeglass, sass) {
-  let all = assetURI(eyeglass, sass);
-  all = merge(all, normalizeURL(eyeglass, sass));
-  all = merge(all, version(eyeglass, sass));
-  all = merge(all, fs(eyeglass, sass));
-  return all;
+const allEyeglassFunctions: EyeglassFunctions = function(eyeglass, sass): FunctionDeclarations {
+  return Object.assign({},
+    assetURI(eyeglass, sass),
+    normalizeURL(eyeglass, sass),
+    version(eyeglass, sass),
+    fs(eyeglass, sass)
+  );
 };
+export default allEyeglassFunctions;
```

Comment:
There is! All the function modules return a consistent interface of `EyeglassFunctions`. By setting this local constant to the type and assigning it, I can move a potential compiler error for this code being malformed to this file. If I just return the function, it will be structurally typed against `EyeglassFunctions` later. Because that error would be returned in a different file (and in this case, possibly after performing a lodash `merge`), debugging it would have more overhead in understanding where things went wrong. Whenever possible I try to move my compiler errors to the offending code; like I'm doing here.